CMakeLists.txt

```
@@ -45,6 +45,24 @@ if(POLICY CMP0135)
   cmake_policy(SET CMP0135 NEW)
 endif()
 
+function(FATAL_ERROR_MISSING_ENV)
+  if(WIN32)
+    if(CMAKE_BUILD_TYPE MATCHES "Debug")
+      message(FATAL_ERROR "Did you download the build environment via ´${CMAKE_SOURCE_DIR}/tools/windows_buildenv.bat´?")
```

```suggestion
      message(FATAL_ERROR "Did you download the Mixxx build environment using ´${CMAKE_SOURCE_DIR}/tools/windows_buildenv.bat´?")
```
The suggested wording makes it maybe more clear, that the buildenv is something, that you get from the Mixxx project. What do you think?